### PR TITLE
transport package with factory pattern

### PR DESCRIPTION
**Pull request by alibyt110**

### CODE REVIEW

File: Examples/FactoryPattern/CargoTransport/Impliment/AirTransport.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+            Console.WriteLine("Transport Air Delivery");
+        }
+    }
+
+    public class AirTransportFactory : TransportFactory
+    {
+        AirTransport airInstance;   
+        public override Transport CreateTransport()
+        {
+            airInstance = new AirTransport();
```
Comment (mohammadKarimi) [inline Examples/FactoryPattern/CargoTransport/Impliment/AirTransport.cs:23]: it must be single instance.

File: Examples/FactoryPattern/CargoTransport/Impliment/ShipTransport.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        {
+            Console.WriteLine($"Transport Ship Delivery from {_origin} to {_destination}");
+        }
+    }
+    public class ShipTransportFactory : TransportFactory
+    {
+        ShipTransport airInstance;
+        public override ShipTransport CreateTransport()
+        {
+            airInstance = new ShipTransport(string.Empty, string.Empty);
```
Comment (mohammadKarimi) [inline Examples/FactoryPattern/CargoTransport/Impliment/ShipTransport.cs:30]: airInstance ?

**Comment by alibyt110** [comment]
@mohammadKarimi 
please review truck

**Comment by alibyt110** [comment]
Dear Mohammad Please I tried again to implement the second part, please check again if possible

**Pull request closed without merging**